docs/site/src/pages/style/color/Color.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import * as colors from 'material-ui/styles/colors';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import { getContrastRatio } from 'material-ui/styles/colorManipulator';
+
+const mainColors = [
+  'Red', 'Pink', 'Purple', 'Deep Purple', 'Indigo', 'Blue', 'Light Blue',
+  'Cyan', 'Teal', 'Green', 'Light Green', 'Lime', 'Yellow', 'Amber', 'Orange',
+  'Deep Orange',
+];
+const neutralColors = ['Brown', 'Grey'];
+
+export const styleSheet = createStyleSheet('colors', (theme) => ({
+  name: {
+    display: 'block',
```

Comment:
Use a `div` over a `span` if you need the display block layouting. Then you can remove that code.